packages/data-broker/src/DataBroker.js

```diff
@@ -246,9 +246,12 @@ export class DataBroker {
     if (countDistinct(dataSources, 'service_type') > 1) {
       throw new Error('Cannot pull metadata for data sources belonging to different services');
     }
-    const service = this.createService(dataSources[0].service_type);
+    const { config: dataGroupConfig, service_type: serviceType } = dataSources[0];
+
+    const optionsWithConfig = { ...options, dataServices: [dataGroupConfig] };
+    const service = this.createService(serviceType);
```

Comment:
Why is this needed?

Reply:
Not sure which line you're referring to, but the config is required to get isDataRegional value for creating api.